test/parallel/test-stream2-httpclient-response-end.js

```diff
@@ -11,7 +11,9 @@ const server = http.createServer(function(req, res) {
     let data = '';
     res.on('readable', common.mustCall(function() {
```

Comment:
So now AFAICT if we want to just solve this, should simply change `readable` to `data`.
Unless we think there's a bug in here somewhere